### ISSUE TITLE
Use :focus-visible to improve focus style

### ISSUE THREAD

**Issue by webzhao** [clair-design/clair#28]
From [`:focus-visible` MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible):

> The `:focus-visible` pseudo-class applies while an element matches the `:focus` pseudo-class and the UA determines via heuristics that the focus should be made evident on the element. (Many browsers show a “focus ring” by default in this case.)

See also:

1. [Keyboard-Only Focus Styles](https://css-tricks.com/keyboard-only-focus-styles/)
2. [`:focus-visible` and backwards compatibility](https://css-tricks.com/focus-visible-and-backwards-compatibility/)

**Comment by AngusFu** [comment]
https://www.npmjs.com/package/postcss-focus-within
